compiler/codegen/expr.cpp

```diff
@@ -4636,6 +4636,17 @@ DEFINE_PRIM(PRIM_SET_DYNAMIC_END_COUNT) {
       ret = rcall->codegen();
 }
 
+DEFINE_PRIM(PRIM_GPU_KERNEL_LAUNCH) {
+
+  std::vector<GenRet> args;
+  for_actuals(actual, call) {
+    args.push_back(actual->codegen());
+  }
+
+  // TODO: make this emit what you want
```

Comment:
Probably should remove this comment? Maybe replace it with a comment that says that we should probably adjust and/or document the arguments for PRIM_GPU_KERNEL_LAUNCH but for now they just go directly to cuLaunchKernel.